e2e/install/kustomize/operator_test.go

```diff
@@ -69,6 +69,10 @@ func TestOperatorBasic(t *testing.T) {
 		Expect(operatorPod.Spec.Containers[0].SecurityContext.AllowPrivilegeEscalation).To(Equal(kubernetes.DefaultOperatorSecurityContext().AllowPrivilegeEscalation))
 
 		Eventually(Platform(ns)).ShouldNot(BeNil())
+		platform := Platform(ns)()
+		Expect(platform.Spec.Build.Registry).ShouldNot(BeNil())
+		Expect(platform.Spec.Build.Registry.Address).To(Equal("kind-registry:5000"))
```

Comment:
I think we have an environment variable controlling that value, `KIND_REGISTRY`. I'd say that, for portability reasons, we should that instead of a constant.

Reply:
Will do, for now I am just trying to check if KEP-1755 feature really works and something is actually extracted from `local-registry-hosting` configmap.